spec/pluralrules.html

```diff
@@ -53,6 +53,82 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-GetOperands" aoid="GetOperands">
+       <h1>GetOperands (x)</h1>
+       <p>
+         When the GetOperands abstract operation is called with argument _x_ (which must be a finite Number value or a String representation of it), it returns an Object value with the following list of plural operands used by plural rules functions.
+       </p>
+
+       <emu-table id="table-plural-operands">
```

Comment:
we should probably move the record table right after the algo instead of above it.